.drone.yml

```diff
@@ -17,10 +17,19 @@ services:
       DEFAULT_REGION: eu-west-2
 
 steps:
+  - name: setup localstack
+    image: quay.io/ukhomeofficedigital/localstack # need curl which is not in alpine
+    commands:
+      - ./config/localstack/setup-queues.sh
```

Comment:
after all that, is it really that easy?

Reply:
oh, because its in STEPS and not Services..we have access to it?

Reply:
Services expose themselves through their name, so we can make sure it's healthy and then just create the queue. Bit upset that I have to have a step for it, but yes, simple really...